config/locales/hyrax.en.yml

```diff
@@ -257,13 +257,21 @@ en:
       workflow_roles:
         header: Workflow Roles
         index:
+          current_group_roles: Current Group Roles
           current_roles: Current Roles
+          current_user_roles: Current User Roles
           delete:
             confirm: Are you sure?
           header:
+            display_name: Display Name
+            email: Email
+            group_name: Group Name
+            group_roles: Group Roles
             roles: Roles
             user: User
-          new_role: Assign Role
+            user_roles: User Roles
```

Comment:
remove line 272-274 for this and all other languages

Reply:
Since all the line numbers don't exactly line up throughout the languages files, would it be easier to identify which keys and values need to be deleted instead of line numbers?

Reply:
removed lines 272-274 for all languages

Reply:
3 lines removed starting at user_role: